src/main/kotlin/racing/domain/RacingGame.kt

```diff
@@ -0,0 +1,55 @@
+package racing.domain
+
+import racing.controller.GameController
+import racing.controller.RacingController
+import racing.view.GameInfo
+
+/*
+car list 를 멤버로 가지며
+이동 로직(랜덤 숫자로 이동) 에 대한 처리를 함
+ */
+class RacingGame(private val gameInfo: GameInfo, private val controller: GameController = RacingController()) {
+    private val cars: List<CarModel>
+    private var currentRound: Int = 0
+
+    init {
+        val carNames = gameInfo.carNames
+        val cars = carNames.split(DEFAULT_DELIMITER)
+        this.cars =
+            List(cars.size) { index ->
+                CarModel(inputName = cars[index], controller = controller)
+            }
+    }
```

Comment:
init 블럭에서 초기화하지 않고, 바로 프로퍼티 초기화하시면 어떨까요? `map`을 활용해볼 수 있을 것 같아요!
```suggestion
class RacingGame(private val gameInfo: GameInfo, private val controller: GameController = RacingController()) {
    private cars = gameInfo.carNames
            .split(DEFAULT_DELIMITER)
            .map { CarModel(inputName = it, controller = controller) }
    private var currentRound: Int = 0
```